src/imitation/scripts/train_adversarial.py

```diff
@@ -36,73 +38,91 @@ def save(trainer, save_path):
-          plots.
-      n_episodes_eval: The number of episodes to average over when calculating
-          the average ground truth reward return of the final policy.
-      checkpoint_interval: Save the discriminator and generator models every
-          `checkpoint_interval` epochs and after training is complete. If <=0,
-          then only save weights after training is complete.
-      interactive: Figures are always saved to `output/*.png`. If `interactive`
-        is True, then also show plots as they are created.
-      expert_policy (BasePolicy or BaseRLModel, optional): If provided, then
-          also plot the performance of this expert policy.
-      init_trainer_kwargs: Keyword arguments passed to `init_trainer`,
-        used to initialize the trainer.
+    _seed: Random seed.
+    env_name: The environment to train in.
+    log_dir: Directory to save models and other logging to.
+
+    n_epochs: The number of epochs to train. Each epoch consists of
+      `n_disc_steps_per_epoch` discriminator steps followed by
+      `n_gen_steps_per_epoch` generator steps.
+    n_gen_steps_per_epoch: The number of generator update steps during every
+      generator epoch.
+    n_disc_steps_per_epoch: The number of discriminator update steps during
+      every training epoch.
+
+    plot_interval: The number of epochs between each plot. (If nonpositive,
+      then plots are disabled).
+    n_episodes_plot: The number of episodes averaged over when
+      calculating the average episode reward of a policy for the performance
+      plots.
+    expert_policy_plot (BasePolicy or BaseRLModel, optional): If provided,
+      then also plot the performance of this expert policy.
+    show_plots: Figures are always saved to `output/*.png`. If `show_plots`
+      is True, then also show plots as they are created.
+
+    ray_tune_interval: The number of epochs between calls to `ray.tune.track`.
+      If nonpositive, disables ray tune. Otherwise, enables hooks
+      that call `ray.tune.track` to track the imitation policy's mean episode
+      reward over time. The script will crash unless `ray.tune` was
+      externally initialized.
+
+    n_episodes_eval: The number of episodes to average over when calculating
+      the average ground truth reward return of the imitation policy for return
+      and for `ray.tune.track`.
+    checkpoint_interval: Save the discriminator and generator models every
+      `checkpoint_interval` epochs and after training is complete. If <=0,
+      then only save weights after training is complete.
+    init_trainer_kwargs: Keyword arguments passed to `init_trainer`,
+      used to initialize the trainer.
 
   Returns:
-      results: A dictionary with two keys, "mean" and "std_err". The
-          corresponding values are the mean and standard error of
-          ground truth episode return for the imitation learning algorithm.
+    A dictionary with the following keys: "ep_reward_mean" and
```

Comment:
Normally you have "and" as the penultimate entry in a list, unless you want to emphasize that "ep_reward_mean" and "ep_reward_std_err" are paired.